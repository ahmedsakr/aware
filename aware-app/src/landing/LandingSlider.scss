.slide-item {
    max-width: 100%;
    min-width: 100%;
    max-height: 99vh;
    min-height: 99vh;    
}