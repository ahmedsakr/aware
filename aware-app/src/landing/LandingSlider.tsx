import React from 'react';
import '../../node_modules/bootstrap/dist/css/bootstrap.css'
import './LandingSlider.scss'

import Slider, {Settings}from 'react-slick'
import 'slick-carousel/slick/slick-theme.css';
import 'slick-carousel/slick/slick.css';

type LandingSliderProps = {};
type LandingSliderState = {};

export default class LandingSlider extends React.Component<LandingSliderProps, LandingSliderState>{
    render(): JSX.Element {
        let settings: Settings = {
            dots: false,
            infinite: true,
            autoplay: true,
            fade: true,
            slidesToShow: 1,
            slidesToScroll: 1,
            autoplaySpeed: 5000,
            speed: 1000,
            lazyLoad: 'ondemand'
        };
        return (
            <div>
                <Slider {...settings}>
                    <div><img className='slide-item' src={process.env.PUBLIC_URL + "/shrek.jpg"} alt="Shrek" /></div>
                    <div><img className='slide-item' src={process.env.PUBLIC_URL + "/test1.jpg"} alt="Image 1" /></div>
                    <div><img className='slide-item' src={process.env.PUBLIC_URL + "/test2.jpg"} alt="Image 2" /></div>
                    <div><img className='slide-item' src={process.env.PUBLIC_URL + "/test3.jpg"} alt="Image 3" /></div>
                    <div><img className='slide-item' src={process.env.PUBLIC_URL + "/test4.jpg"} alt="Image 4" /></div>
                </Slider>
            </div>
        );
    }
}
