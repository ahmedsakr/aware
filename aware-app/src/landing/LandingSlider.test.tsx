import React from 'react';
import { shallow } from 'enzyme';
import '../testSetup'

import LandingSlider from './LandingSlider';

describe('LandingSlider', () => {
    it('should render correctly', () => {
        const component = shallow(
            <LandingSlider/>
        );

        expect(component).toMatchSnapshot();
    });
});