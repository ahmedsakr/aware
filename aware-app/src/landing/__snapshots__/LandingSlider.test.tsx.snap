version https://git-lfs.github.com/spec/v1
oid sha256:aacf32c008106bae67f861cf8e1e72b438b55ac753f2edb78f994702521cfd50
size 893
